templates/config/i18n-tasks.yml

```diff
@@ -115,16 +116,16 @@ search:
 #     formality: prefer_less
 ## Do not consider these keys missing:
 # ignore_missing:
-# - 'errors.messages.{accepted,blank,invalid,too_short,too_long}'
-# - '{devise,simple_form}.*'
+#   - 'errors.messages.{accepted,blank,invalid,too_short,too_long}'
```

Comment:
Similarly here, this doesn't make commenting out easier. The extra identation is not needed in YAML.

Reply:
This part is where I've made fixes related to the consistency of code style rather than ease of commenting out/in.

Yes, you're right, this indentation is not required.
But as we can see [here](https://github.com/glebm/i18n-tasks/blob/deba8884a20b05e4df65ca99ab3a5f4887815a47/templates/config/i18n-tasks.yml#L17-L21) or [here](https://github.com/glebm/i18n-tasks/blob/deba8884a20b05e4df65ca99ab3a5f4887815a47/templates/config/i18n-tasks.yml#L77-L81) etc, array lists are basically written with the indentation in this YAML file, so I just thought it would be better to make it consistent with them as well.